### PR TITLE
Code Quality: Move new tab button to TabBar itself

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Summary

MainPage doesn't have to hold the button, rather than that TabBar should be responsible to that button. Also less code on MainPage code-nehind will help us in the future when we actually decided to remove ShellPanesPage (aka PaneHolderPage previsouly)

### Resolved / Related Issues

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #

### Steps used to test these changes

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Open Files app
2. Create a new tab
3. See no crash or hang

### CODE REVIEW

**Comment by yaira2** [comment]
I think I'm the one who did it this way, but I don't remember why. It might be worth looking at earlier commits to see why it was placed in MainPage.

**Comment by 0x5bfa** [comment]
I went back to [blame/Files/Views/MainPage/253bd66ca894cf15ea4b0c012c5957b08bf388aa](https://github.com/files-community/Files/blame/253bd66ca894cf15ea4b0c012c5957b08bf388aa/Files/Views/MainPage.xaml) but I guess why this is here is we had also vertical tab button here. Then [you just removed this button](https://github.com/files-community/Files/pull/9786). Do you have an idea where you did talk about this?

And in my opinion, removing unnecessary xaml from MainPage would help us adding more features and TabBar has this button's responsibilility and MainPage doesn't have anything to do with this button.

**Comment by yaira2** [comment]
Agreed, aside from the vertical tab flyout, I'm not sure why we implemented it this way.

**Comment by 0x5bfa** [comment]
The first person who added Add new button to here https://github.com/files-community/Files/pull/5214 but I understand it's normal thing to move it to MainPage because AddNewTab function was in MainPageViewModel

**Comment by yaira2** [comment]
Can you resolve the merge conflict?

File: src/Files.App/UserControls/TabBar/TabBar.xaml.cs [hunk 1]
```diff
@@ -22,6 +22,10 @@ public sealed partial class TabBar : BaseTabBar
 
 		private TabViewItem? hoveredTabViewItem;
 
+		private bool _lockDropOperation = false;
+
+		//private string[] _droppableArchiveTypes = { "zip", "rar", "7z", "tar" };
```
Comment (yaira2) [inline src/Files.App/UserControls/TabBar/TabBar.xaml.cs:27]: If we aren't using this, I think it can be removed.

**Review by yaira2 — DISMISSED** [review]

**Pull request merged**